ddp-workspace/projects/ddp-dsm-ui/src/app/FON/components/register-patients-modal/register-patients-modal.component.html

```diff
@@ -84,7 +84,7 @@
     </aside>
   </main>
   <footer>
-    <button class="btn_secondary" (click)="closeDialog()">Cancel</button>
-    <button class="btn_primary" [disabled]="addPatientForm.invalid" (click)="registerPatient()">Register</button>
+    <button class="btn_secondary" [ngClass]="{'disappear': uploading}" (click)="closeDialog()">Cancel</button>
```

Comment:
We can not cancel uploading ? 
Is it correct?

Reply:
Yes, right